requirements.txt

```diff
@@ -41,3 +41,6 @@ django-ptvsd-debug==1.0.3
 django-lti-auth==1.0.0
 django-webpack-loader==0.6.0
 django-csp==3.5
+
+# Remove this when upgrading to Python 3.7
+typing_extensions==3.7.4.1
```

Comment:
This can be removed, right?

Reply:
Ok, I've removed this. They're still in the typing_extensions package but this is included by default now.

Reply:
Looks like it's still there? Did I miss something?

Reply:
Yeah not sure why that was missed, it's there now.